vcd/resource_vcd_catalog.go

```diff
@@ -0,0 +1,133 @@
+package vcd
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/vmware/go-vcloud-director/govcd"
+	"log"
+)
+
+func resourceVcdCatalog() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceVcdCatalogCreate,
+		Delete: resourceVcdCatalogDelete,
+		Read:   resourceVcdCatalogRead,
+
+		//resource "vcd_catalog" "OperatingSystems" {
```

Comment:
Please remove these comments. They add nothing to the code.

Reply:
done